lxdock/conf/schema.py

```diff
@@ -12,6 +12,7 @@
     'lxc_config': {Extra: str},
     'mode': In(['local', 'pull', ]),
     'privileged': bool,
+    'profiles': [str, ],
```

Comment:
I think that profile names have to be real LXD identifiers. Could you use `'profiles': [LXDIdentifier(), ],` instead?